packages/core/admin/admin/src/pages/ProfilePage/tests/index.test.js

```diff
@@ -68,9 +66,80 @@ describe('ADMIN | Pages | Profile page', () => {
   });
 
   it('should display username if it exists', async () => {
-    render(App);
+    setup();
+    await waitFor(() => {
+      expect(screen.getByText('yolo')).toBeInTheDocument();
+    });
+  });
+
+  it('should display the change password section and all its fields', async () => {
+    const { getByRole, getByTestId, queryByTestId } =  setup();
+
+    await waitFor(() => {
+      expect(queryByTestId('loader')).not.toBeInTheDocument();
+    });
+
+    const changePasswordHeading = getByRole('heading', {
+      name: 'Change password'
+    });
+
+    await waitFor(() => {
+      expect(changePasswordHeading).toBeInTheDocument();
+    });
+
+    await waitFor(() => {
+      expect(getByTestId('test-current-password-input')).toBeInTheDocument()
+    });
+
+    await waitFor(() => {
+      expect(getByTestId('test-new-password-input')).toBeInTheDocument()
+    });
+
+    await waitFor(() => {
+      expect(getByTestId('test-confirmed-password-input')).toBeInTheDocument()
+    });
```

Comment:
I think you don't have to use waitFor here every time:

```suggestion
    expect(getByRole('heading', {
      name: 'Change password'
    })).toBeInTheDocument();

    expect(getByTestId('test-current-password-input')).toBeInTheDocument()

    expect(getByTestId('test-new-password-input')).toBeInTheDocument()

    expect(getByTestId('test-confirmed-password-input')).toBeInTheDocument()
```

That makes the test easier to read and easier to spot test failures.

Reply:
done